src/test/java/seedu/address/testutil/CompanyUtil.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.addcommands.AddCompanyCommand;
+import seedu.address.model.company.Company;
+
+/**
+ * A utility class for Company.
+ */
+public class CompanyUtil {
+
+    /**
+     * Returns an add command string for adding the {@code company}.
+     */
+    public static String getAddCommand(Company company) {
+        return AddCompanyCommand.COMMAND_WORD + " c " + getcompanyDetails(company);
+    }
+
+    /**
+     * Returns the part of command string for the given {@code Company}'s details.
+     */
+    public static String getcompanyDetails(Company company) {
+        StringBuilder sb = new StringBuilder();
+        sb.append(PREFIX_NAME + company.getCompanyName().fullName + " ");
+        sb.append(PREFIX_PHONE + company.getCompanyPhone().value + " ");
+        sb.append(PREFIX_EMAIL + company.getCompanyEmail().value + " ");
+        sb.append(PREFIX_DESCRIPTION + company.getDescription().value + " ");
+        company.getTags().stream().forEach(
+            s -> sb.append(PREFIX_TAG + s.tagName + " ")
+        );
+        return sb.toString();
+    }
+
+    //    /**
+    //     * Returns the part of command string for the given {@code EditcompanyDescriptor}'s details.
+    //     */
+    // public static String getEditcompanyDescriptorDetails(EditCompanyDescriptor descriptor) {
+    //     StringBuilder sb = new StringBuilder();
+    //     descriptor.getName().ifPresent(name -> sb.append(PREFIX_NAME).append(name.fullName).append(" "));
+    //     descriptor.getPhone().ifPresent(phone -> sb.append(PREFIX_PHONE).append(phone.value).append(" "));
+    //     descriptor.getEmail().ifPresent(email -> sb.append(PREFIX_EMAIL).append(email.value).append(" "));
+    //     descriptor.getDescription().ifPresent(
+    //          description -> sb.append(PREFIX_DESCRIPTION).append(description.value).append(" "));
+    //     if (descriptor.getTags().isPresent()) {
+    //         Set<Tag> tags = descriptor.getTags().get();
+    //         if (tags.isEmpty()) {
+    //             sb.append(PREFIX_TAG);
+    //         } else {
+    //             tags.forEach(s -> sb.append(PREFIX_TAG).append(s.tagName).append(" "));
+    //         }
+    //     }
+    //     return sb.toString();
+    // }
```

Comment:
If the test is not used, perhaps we can consider deleting it?

Reply:
This could be useful after we implementing EditCompanyCommand and related functionalities. Just treat it as a placeholder for now.

Reply:
I'm not sure if we should keep such a huge chunk of code commented out. I feel like we can uncomment it and then put a comment on line 43 to say it's a placeholder? What do you think?

Reply:
Cuz we haven't implement company related content for EditCommand. Classes or methods needed for this test is not yet implemented, so for now it doesn't work even if we uncomment it.